### DISCUSSION TITLE
Please, add support for images

### DISCUSSION THREAD

**Discussion by GrigoriiPrudnikov** [zed-industries/zed#18544] [Feature Requests]
Please, add support for images. I want to open images it tab inside zed (like vscode), instead of having 2 windows.

**Comment by htor** [comment]
It already supports images. Just try `File > Open` or drag the image into a buffer.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
